source/fe/fe_system.cc

```diff
@@ -1296,16 +1295,15 @@ FESystem<dim,spacedim>::compute_fill (
       Threads::TaskGroup<> task_group;
       for (unsigned int base_no=0; base_no<this->n_base_elements(); ++base_no)
         task_group
-        += Threads::new_task (std_cxx11::function<void ()>(std_cxx11::bind (&FESystem<dim,spacedim>::template compute_fill_one_base<dim_1>,
+        += Threads::new_task (std_cxx11::function<void ()>(std_cxx11::bind (&FESystem<dim,spacedim>::compute_fill_one_base<dim_1>,
```

Comment:
Good spot, no reason to change it (it did compile on three different compilation units for me even without, but the second variant should be safer).